scripts/gsc-indexing-audit.py

```
@@ -0,0 +1,290 @@
+            print(f"GSC error: {e}", file=sys.stderr)
+            break
+        rows = res.get("rows", [])
+        out.extend(rows)
+        if len(rows) < 25000 or len(out) >= limit:
+            break
+        start_row += 25000
+    return out
+
+
+def pattern_aggregation(gsc):
+    print(f"\n=== PERFORMANCE BY URL PATTERN (current 28d vs prior 28d) ===")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)